### PR TITLE
Massive Refactor - 3.0.0

### PR DESCRIPTION
**Pull request by itsthejoker**
This PR strips out the Archivist and the OCR bot, then all the duplicate code now stored in github.com/transcribersofreddit/tor_core. Tested and working.

### CODE REVIEW

File: tor/main.py [hunk 1]
```diff
@@ -1,86 +1,48 @@
...
-
-    except KeyboardInterrupt:
-        logging.info('User triggered shutdown. Shutting down.')
-        sys.exit(0)
-
-    except Exception as e:
-        explode_gracefully('u/ToR', e, tor)
-
+    build_bot('bot', __version__, full_name='u/ToR')
+    run_until_dead(run)
```
Comment (TheLonelyGhost) [inline tor/main.py]: I really like how much simpler this is! 👍

Comment (itsthejoker) [inline tor/main.py]: Isn't it fantastic‽ Wait until you see how much code I get to strip out of OCR and Archivist!

File: requirements/base.txt [hunk 2]
```diff
@@ -1,9 +1,11 @@
-praw==4.4.0
+praw==5.0.1
```
Comment (itsthejoker) [inline requirements/base.txt]: I installed it and nothing seemed to explode; this is one of those "keep a close eye on it" things.

**Comment by TheLonelyGhost** [comment]
Just to be clear, this is ready to merge? @itsthejoker

**Comment by itsthejoker** [comment]
It is, I'm still pulling code from it to finish up the other two bots. Almost ready -- just making sure I didn't miss anything.

**Pull request merged**